File: C/24BCSH93/1st Semester/assignment6/question4.c
#include <stdio.h>

int main() {
    int num, i, j;
    printf("Enter the number: ");
    scanf("%d", &num);

    for (i = 1; i <= num; i++) {
        for (j = 1; j <= i; j++) {
            printf("%d ", i % 2);
        }
        printf("\n");
    }
    return 0;
}

File: Java/Sec-A/assignment1/question4.java
//  Q4. Write a Java program to test the inputted number is prime or composite.

import java.util.Scanner;

class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter a number: ");
        int num = sc.nextInt();

        for (int i = 2; i <= num / 2; i++) {
            if (num % i == 0) {
                System.out.println(num + " is a composite number");
                return;
            }
        }
        System.out.println(num + " is a prime number");
    }
}
